client/webserver/site/src/css/wallets.scss

```diff
@@ -92,3 +92,9 @@ table.wallets {
   cursor: pointer;
   color: #a8a8a8;
 }
+
+#copyAlert {
+  margin-left: 9px;
```

Comment:
With 9x it's right up against the `div` for me.  Maybe 12?

Reply:
12 looks good.